s4e-web/cypress/page-objects/map/map.po.ts

```diff
@@ -16,8 +16,8 @@ export class Map extends Core {
     getLogoutBtn: () => cy.get("a").contains("Wyloguj"),
 
     // Date selection
-    getDateChangeBtn: () => cy.get(".timeline__changedate"),
-    getYearSelectionBtn: () => cy.get(".owl-dt-control-period-button"),
+    getDateChangeBtn: () => cy.get(".timeline__changedate"),//calendar icon
+    getYearSelectionBtn: () => cy.get(".owl-dt-control-period-button"), //month year on open calendar
```

Comment:
Please replace all names of keys in dict to more meaningful instead of writing comments.